readme.md

```diff
@@ -24,6 +30,14 @@ github "sindresorhus/LaunchAtLogin"
 
 Add a new ["Run Script Phase"](http://stackoverflow.com/a/39633955/64949) **below** "Embed Frameworks" in "Build Phases" with the following:
 
+#### SwiftPM
+
+```sh
+"${PROJECT_DIR}/Carthage/Build/Mac/LaunchAtLogin.framework/Resources/copy-helper-swiftpm.sh"
```

Comment:
This is still pointing to the Carthage directory.

Reply:
Fixed. To test how it works you can use:
```
.package(url: "https://github.com/SergeyKuryanov/LaunchAtLogin", .branch("4_swift_package_manager_support"))```